test/helpers/builders.js

```diff
@@ -0,0 +1,24 @@
+// Copyright IBM Corp. 2014,2016. All Rights Reserved.
+// Node module: strong-remoting
+// This file is licensed under the Artistic License 2.0.
+// License text available at https://opensource.org/licenses/Artistic-2.0
+
+'use strict';
+
+exports.remotingDate = function(val) {
+  return {
+    // NOTE(bajtos) The current implementation of strong-remoting
+    // calls `.toString()` which returns a date value like this:
+    //     Sat Jan 01 2000 00:00:00 GMT+0100 (CET)
+    // I think we should use `.toJSON()` instead, which returns concise
+    //     1999-12-31T23:00:00.000Z
+    '$data': new Date(val).toString(),
```

Comment:
Yeah this should not use `toString()`. Is there a way to support both? Maybe a format setting somewhere. Not in this PR though.